benchmarks/fastify-big-json.js

```diff
@@ -20,17 +18,21 @@ const opts = {
   }
 }
 
-function Eng (id, title, emp) {
+function Employee ({ id = null, title = null, employer = null } = {}) {
   this.id = id
   this.title = title
-  this.employer = emp
+  this.employer = employer
 }
 
-fastify.get('/', opts, function (request, reply) {
+fastify.get('/', opts, (request, reply) => {
```

Comment:
can you please restore all the function definition that were `function () {}`?

Reply:
@mcollina -Just for my personal understanding; what's wrong with the arrow function? -Is there any scoping requirement?

Reply:
nothing. It's not needed to support standard, and using them _might_ have a slight influence in performance. So I prefer to use normal functions in benchmarks.

Reply:
They are, however, used in other places, such as https://github.com/fastify/benchmarks/blob/master/benchmarks/express-route-prefix.js -Not fair to use them elsewhere if you think arrow functions might affect performance.

Reply:
Those have been contributed by doug, so if he thinks it's ok then it's ok. I'm just asking **to not change them** for linting reasons.

If you want to change them, open a different PR.

Reply:
Oh, I was not changing them for linting reasons, they were part of the overall PR, hence the title: `ESLint, updated README.md, minor code clean-up.`

Reply:
it's not "minor" to me.

Reply:
The express-route-prefix was from before I started contributing here :) it should use a plain function imo

Reply:
Looks like Koa and Restify are using srrow functions. Probably a PR to make them consistent would be in order.

Reply:
@dougwilson -Yes, that is already in my plan as part of another PR once this one is merged.